modules/contrib/src/entities/Chat.ts

```diff
@@ -30,20 +29,22 @@ export const Chat = (): Entity => {
     fontSize: 16,
     color: 0xFFFF33,
     dynamic: (t: HTMLText) => {
-      chatIsOpen ? t.text = `${chatBuffer.join("")}|` : t.text = ""
+      const textToRender = chatBuffer.map((char) => (char === " ") ? "&nbsp;" : char).join("");
+      chatIsOpen ? t.text = `${textToRender}|` : t.text = ""
```

Comment:
fixed bug where typing multiple `" "` would not appear in the chat, but would still be in buffer